### PR TITLE
Add local/util.sh

### LINKED ISSUE #25452: Add local/util.sh for local cluster e2e testing
If I specify `KUBERNETES_PROVIDER=local` to run e2e testing, an error is happening immediately:

```
2016/05/11 11:00:00 e2e.go:194: Running: get status
/home/kubernetes/hack/e2e-internal/e2e-status.sh: line 30: prepare-e2e: command not found
```

Comparing to other providers, cluster/local/util.sh should be there and include prepare-e2e.

### PR DESCRIPTION
**Pull request by gouyang**
Fix #25452

### CODE REVIEW

**Comment by davidopp** [comment]
@gouyang This PR looks fine, but I was not aware of "local" cluster type. Can you point me to the documentation that mentions it? Then I will LGTM the PR. Thanks!

**Comment by jeffvance** [comment]
@davidopp Here an example of local clusters being referenced in the e2e docs:
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/e2e-tests.md#local-clusters

And here's a PR that was merged a while back to support local clusters better:
https://github.com/kubernetes/kubernetes/commit/61cc821506af2de05c2c902960dd64b9cca16188

**Comment by davidopp** [comment]
LGTM

**Comment by gouyang** [comment]
@davidopp @jeffvance Except the example in e2e doc, I was setting up the cluster via contrib/ansible on local machines, so the cluster is local.

*No test results found` for jenkins integration testing  should be okay.

**Comment by jeffvance** [comment]
@gouyang I've been testing with local-up-cluster and haven't tried the ansible way yet...

**Comment by ixdy** [comment]
@k8s-bot unit test this, issue #IGNORE

File: cluster/local/util.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```
Comment (ixdy) [inline cluster/local/util.sh]: this needs the copyright boilerplate from `master/hack/boilerplate/boilerplate.sh.txt`.

File: cluster/local/util.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+# Perform preparations required to run e2e tests
+function prepare-e2e() {
+  echo "Local doesn't need special preparations for e2e tests" 1>&2
+
+}
+
```
Comment (ixdy) [inline cluster/local/util.sh]: please remove extraneous newlines

**Comment by goltermann** [comment]
@gouyang  This PR has the v1.3 milestone tag applied, but hasn't been touched in a week or more. We will cut Beta and branch on June 10. Please either close, finish off, get an LGTM or rebase this as needed to make sure it makes its way into 1.3.

**Comment by gouyang** [comment]
@ixdy Thank you for the review, I've updated.

**Comment by ixdy** [comment]
Thanks for the fixes, lgtm

**Pull request merged**